ddtrace/contrib/internal/langgraph/patch.py

```diff
@@ -168,8 +180,12 @@ async def _astream():
                 integration.llmobs_set_tags(span, args=args, kwargs=kwargs, response=response, operation="node")
                 span.finish()
                 break
-            except Exception:
-                span.set_exc_info(*sys.exc_info())
+            except Exception as e:
+                if LangGraphParentCommandError is None or not isinstance(e, LangGraphParentCommandError):
+                    # LangGraph will throw this error to communicate that a Command operation has exited a child node
+                    # to navigate to the parent graph. This error is caught in the LangGraph framework, and we should
+                    # not mark it as a runtime error.
```

Comment:
```suggestion
                    # This error is caught in the LangGraph framework, we shouldn't mark it as a runtime error.
```
Let's also add this to the other instance checks for LangGraphParentCommandError